pkg/api/dashboard_snapshot_test.go

```diff
@@ -154,7 +154,8 @@ func TestDashboardSnapshotApiEndpoint(t *testing.T) {
 			Convey("Should gracefully delete local snapshot when remote snapshot has already been removed", func() {
 				loggedInUserScenarioWithRole("When calling DELETE on", "DELETE", "/api/snapshots/12345", "/api/snapshots/:key", m.ROLE_EDITOR, func(sc *scenarioContext) {
 					ts := setupRemoteServer(func(rw http.ResponseWriter, req *http.Request) {
-						rw.Write([]byte(`{"message":"Failed to get dashboard snapshot"}`))
+						_, err := rw.Write([]byte(`{"message":"Failed to get dashboard snapshot"}`))
+						So(err, ShouldBeNil)
```

Comment:
The reason test is panic:ing is probably because of this line is made inside the http handler. You maybe have to declare a variable to assign the error and verify it at the bottom of the test?